features/zoek-met-burgerservicenummer/in-onderzoek.feature

```diff
@@ -0,0 +1,195 @@
+    Gegeven de persoon met burgerservicenummer '000000152' heeft een 'reisdocument' met de volgende gegevens
+    | naam                                                                    | waarde                    |
+    | soort reisdocument (35.10)                                              | PN                        |
+    | nummer reisdocument (35.20)                                             | NE3663258                 |
+    | datum einde geldigheid reisdocument (35.50)                             | 20240506                  |
+    | aanduiding in onderzoek (83.10)                                         | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)                                          | 20230506                  |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | gemeente van inschrijving (09.10) |
+    | 0800                              |
+    Als reisdocumenten wordt gezocht met de volgende parameters
+    | naam                    | waarde                                        |
+    | type                    | ZoekMetBurgerservicenummer                    |
+    | burgerservicenummer     | 000000152                                     |
+    | gemeenteVanInschrijving | 0800                                          |
+    | fields                  | reisdocumentnummer,soort,datumEindeGeldigheid |
+    Dan heeft de response een 'reisdocument' met de volgende gegevens
+    | naam                             | waarde             |
+    | reisdocumentnummer               | NE3663258          |
+    | soort.code                       | PN                 |
+    | soort.omschrijving               | Nationaal paspoort |
+    | datumEindeGeldigheid.type        | Datum              |
+    | datumEindeGeldigheid.datum       | 2024-05-06         |
+    | datumEindeGeldigheid.langFormaat | 6 mei 2024         |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | type                                                    | opmerking                                    |
+    | 123530                  | datum uitgifte Nederlands reisdocument                  | geen veld van reisdocument resource          |
+    | 123540                  | autoriteit van uitgifte Nederlands reisdocument         | geen veld van reisdocument resource          |
+    | 123560                  | datum inhouding/vermissing Nederlands reisdocument      | niet gevraagd veld van reisdocument resource |
+    | 123570                  | aanduiding inhouding/vermissing Nederlands reisdocument | niet gevraagd veld van reisdocument resource |
+
+  Abstract Scenario: '<type>' is in onderzoek en de houder velden wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens 
+    | naam                            | waarde                            |
+    | aanduiding in onderzoek (83.10) | <aanduiding persoon in onderzoek> |
```

Comment:
\<aanduiding persoon in onderzoek> moet zijn \<aanduiding in onderzoek>

Reply:
aangepast